website/docs/using-unleash/compliance/soc2.mdx

```diff
@@ -9,7 +9,7 @@ description: 'SOC2-compliant feature flags at scale with Unleash.'
 
 To get SOC2 certified and maintain your compliance, you must ensure that any system you integrate with, including feature flagging solutions, are also SOC2 certified. Using a homegrown or third-party feature flagging system without SOC2 compliance can compromise your certification and introduce unnecessary risks.
 
-This guide provides an overview of how Unleash features align with SOC2 Type II controls, helping your organization meet its compliance requirements.
+This guide provides an overview of how [Unleash Enterprise](https://www.getunleash.io/enterprise-feature-management-platform) features align with SOC2 Type II controls, helping your organization meet its compliance requirements.
```

Comment:
Happy to go ahead with this change, but technically not _all_ the features we list in the table need Enterprise